developer/tutorial/plugin-customizing-templates-4.md

```diff
@@ -51,7 +51,7 @@ import PropTypes from "prop-types";
 import classnames from "classnames";
 import Blaze from "meteor/gadicc:blaze-react-component";
 import { Template } from "meteor/templating";
-import { registerComponent } from "/imports/plugins/core/layout/lib/components";
+import { registerComponent } from "@reactioncommerce/reaction-components";
```

Comment:
This would require to have reaction-components aliased in plugin's package.json, which is not done automatically (maybe it should?) when creating the plugin via reaction plugins create --name <plugin_name>. I'd leave this as is for the moment and create an issue for reaction-cli instead. Also these code changes should be in sync with https://github.com/reactioncommerce/reaction-example-plugin/blob/master/client/templates/layouts/core.js